src/components/MovieCard.vue

```diff
@@ -38,16 +44,26 @@ export default {
 
   &__title {
     @include font-heading--medium;
-    padding-bottom: 0.45em;
+    color: $text-dark;
+    padding-bottom: 0.25em;
   }
 
-  &__description {
-    @include font-paragraph--small;
+  &__length {
+    @include font-paragraph--the-smallest;
+    text-transform: none;
```

Comment:
I would remove text-transform from the mixin instead - it will be less CSS for the same job